### ISSUE TITLE
Clarification on output files

### ISSUE THREAD

**Issue by danrhouck** [SNL-WaterPower/CACTUS#26]
The User Guide says that, when FieldOutFlag is on, it outputs induced velocity on a 3D Cartesian grid. I'm getting an essentially 2D output as all of the x-coordinates are zero at all times. Is that what it really should be or is this an issue? If an issue, whos?

Additionally, WakeElemData can be output, which appears to be nominally the same except that it is not output on a Cartesian grid. Will implementing the wake grid options (nxgrid, nygrid, nzgrid, etc.) force this output onto a Cartesian grid?

Basically, I'm wondering what the easiest way is to get an output of 3D induced velocities on a Cartesian grid.

**Comment by danrhouck** [comment]
User Guide also says that default x-coordinates are 0 to 0, which explains the output. I have tested to confirm that the grid inputs only affect the FieldData output. I think the confusion came from an unpublished report using CACTUS that has an input file in the appendix. In that report, the FieldOutFlag is not on, but the grid is specified.
